City-of-Helsinki/berth-reservations-admin
# VEN-1213 | Single Customer Page: Show All Leases Regardless of Their Statuses

## Description :sparkles:
- Remove the condition of showing only `PAID` leases

## Issues :bug:

### Closes :no_good_woman:
- [VEN-1213](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1213): Terminate lease doesn't work if the invoice is in error state

### Related :handshake:

## Testing :alembic:

### Automated tests :gear:️
- Updated the corresponding tests

### Manual testing :construction_worker_man:
- Go to one of the customers that have a lease in `ERROR` state, e.g. [Testinen Testinen](http://venepaikka-admin.test.kuva.hel.ninja/customers/UHJvZmlsZU5vZGU6M2RlYjc3M2MtODViMy00ODQ2LTk0ZTItMzJhMjliMGE2YzQ5
)
- In the `VENEPAIKAT` card, you should be able to see the lease in question

## Screenshots :camera_flash:
![image](https://user-images.githubusercontent.com/23040926/111296857-a3993100-8655-11eb-98bd-98c262347ea3.png)

## Additional notes :spiral_notepad:
The [backend ticket](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1214) should be resolved first in order to be able to proceed with terminating the lease.
